physics/sfcsub.F

```diff
@@ -1335,17 +1313,10 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc                  &
         call monitr('aisclm',aisclm,sliclm,snoclm,len)
         call monitr('snoclm',snoclm,sliclm,snoclm,len)
         call monitr('scvclm',scvclm,sliclm,snoclm,len)
-        call monitr('smcclm1',smcclm(1,1),sliclm,snoclm,len)
-        call monitr('smcclm2',smcclm(1,2),sliclm,snoclm,len)
-        call monitr('stcclm1',stcclm(1,1),sliclm,snoclm,len)
-        call monitr('stcclm2',stcclm(1,2),sliclm,snoclm,len)
-!clu [+4l] add smcclm(3:4) and stcclm(3:4)
-        if(lsoil.gt.2) then
-        call monitr('smcclm3',smcclm(1,3),sliclm,snoclm,len)
-        call monitr('smcclm4',smcclm(1,4),sliclm,snoclm,len)
-        call monitr('stcclm3',stcclm(1,3),sliclm,snoclm,len)
-        call monitr('stcclm4',stcclm(1,4),sliclm,snoclm,len)
-        endif
+        do k=1,lsoil
+          call monitr('smcclm1',smcclm(1,k),sliclm,snoclm,len)
```

Comment:
Should we generate smcclm`k` instead of calling it `smcclm1`?

Reply:
yes

Reply:
Can you please make this change?